### PR TITLE
Doing a live demo hi erik

### PR DESCRIPTION
**Pull request by LitKnd**
Very emaningfull comments

### CODE REVIEW

**Comment by erikdarlingdata** [comment]
Working as intended. Please drive through.

**Comment by erikdarlingdata** [comment]
(You might need this open for things to work.)

**Comment by LitKnd** [comment]
We are talking about you

File: sp_LogHunter/sp_LogHunter.sql [hunk 1]
```diff
@@ -364,7 +364,8 @@ BEGIN
             ('Warning ******************'), ('SQL Server should be restarted'), ('Server name is'), ('Could not connect'), ('yielding'), ('worker thread'), ('A new connection was rejected'), 
             ('A significant part of sql server process memory has been paged out'), ('Dispatcher'), ('I/O requests taking longer than'), ('killed'), ('SQL Server could not start'),
```
Comment (LitKnd) [inline sp_LogHunter/sp_LogHunter.sql:365]: THis just isn't right
just kidding

**Comment by erikdarlingdata** [comment]
@LitKnd I knew this MIT license thing was a mistake.

**Pull request merged**